### PR TITLE
Updated the issue template

### PR DESCRIPTION
**Pull request by siddhiiguptaa**
This was a super fun beginner friendly issue! I really enjoyed it. Thank You again.

### CODE REVIEW

**Comment by lohanidamodar** [comment]
@siddhiiguptaa Great work. One more change before we can approve. We have made few changes in the language and also the format of Code of conduct. Please check https://github.com/appwrite/appwrite template and update accordingly.
Thank you.

**Comment by lohanidamodar** [comment]
@siddhiiguptaa Great work. One more change before we can approve. We have made few changes in the language and also the format of Code of conduct. Please check https://github.com/appwrite/appwrite template and update accordingly.
Thank you.

**Comment by siddhiiguptaa** [comment]
> @siddhiiguptaa Great work. One more change before we can approve. We have made few changes in the language and also the format of Code of conduct. Please check https://github.com/appwrite/appwrite template and update accordingly. Thank you.

hello @lohanidamodar Updated the template accordingly, kindly check and let me knw if I can fix anything else!

File: .github/ISSUE_TEMPLATE/bug.yaml
Comment (lohanidamodar) [inline .github/ISSUE_TEMPLATE/bug.yaml]: Please update this field format to make it consistent.

**Review by lohanidamodar — COMMENTED** [review]
@siddhiiguptaa great job 👍🏻  Just a small fix to make it consistent.

File: .github/ISSUE_TEMPLATE/bug.yaml
Comment (siddhiiguptaa) [reply .github/ISSUE_TEMPLATE/bug.yaml]: This was really cool yet again. I am glad I am learning how to make documentations consistent throughout. As a UX Designer, I have read a lot about consistency but only got to implement it today. Thank You. Let me know if there's anything else that's needed to be done. 💯

**Review by lohanidamodar — APPROVED** [review]
Great work @siddhiiguptaa 🚀 Please feel free to look for more PRs you might be interested in helping with on our long list of [Hacktoberfest friendly issues](https://github.com/search?q=org%3Aappwrite+org%3Autopia-php+is%3Aissue+label%3Ahacktoberfest+created%3A%3E2021-01-01+is%3Aopen&type=issues) to help make Appwrite better.

**Comment by christyjacob4** [comment]
THANK YOU! All changes merged 🥳

Please reach out to me on our [Discord server](https://appwrite.io/discord) if you would like to claim your Appwrite swags! As a way of saying thank you, we would also love to invite you to join the Appwrite organization on GitHub. Please share your GitHub username with us on Discord.  

You can accept the invite by visiting https://github.com/orgs/appwrite/invitation. By joining our team, you will officially be an Appwrite maintainer on GitHub.

You can change your membership [visibility settings](https://help.github.com/en/github/setting-up-and-managing-your-github-user-account/publicizing-or-hiding-organization-membership), so your new Appwrite team membership badge will show up on your personal GitHub profile.

Please feel free to look for more PRs you might be interested in helping with on our long list of [Hacktoberfest friendly issues](https://github.com/search?q=org%3Aappwrite+org%3Autopia-php+is%3Aissue+label%3Ahacktoberfest+created%3A%3E2021-01-01+is%3Aopen&type=issues) and help make Appwrite better :)

**Pull request merged**
